src/cloud/config_manager/controllers/server.go

```diff
@@ -218,7 +219,19 @@ func (s *Server) GetConfigForVizier(ctx context.Context,
 	AddDefaultTableStoreSize(tmplValues.PEMMemoryRequest, tmplValues.CustomPEMFlags)
 
 	// Next we inject any feature flags that we want to set for this org.
-	orgID, err := s.getOrgIDForDeployKey(tmplValues.DeployKey)
+	var orgID uuid.UUID
+	// Attempt to get the org from Vizier, otherwise from the DeployKey.
+	if in.VizierID != "" {
```

Comment:
In this part of the logic, I wanted to get some feedback on a different approach where the org information is attempted initially by `GetOrgFromVizier` and if that doesn't work then tried by `getOrgIDForDeployKey`.  
Currently it tries one method and if that doesn't work it wont try the other.

Reply:
I think it might be more robust if we do something like:
- Check if getting org from deployKey works (I would start with this case first, since this is the status quo)
- If deployKey method doesn't work, fallback to using the VizierID.